### PR TITLE
feat: Add Brainerd lab collaboration

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
I pulled the description right off of Beth's lab's description. I think that's okay?

### CODE REVIEW

File: schemas/collaborations.yml [hunk 1]
```diff
@@ -29,7 +30,7 @@ id: "enum(
           'aer-group',
           'wilhelmus-lab',
           'the-policy-lab',
-	  'acadia'
+	        'acadia'
```
Comment (eldu) [inline schemas/collaborations.yml:33]: I think it doesn't like this spacing?

File: schemas/collaborations.yml [hunk 2]
```diff
@@ -46,6 +47,7 @@ department: "enum(
             'computational-psychiatry',
             'sociology',
             'ecology',
+            'ecology-and-evolutionary-biology',
```
Comment (hollandjg) [inline schemas/collaborations.yml:50]: I'm not sure this is quite right – it's what they have on the group website, but the Brown department is the "[Department of Ecology, Evolution and Organismal Biology](https://www.brown.edu/academics/ecology-and-evolutionary-biology/)". Maybe just leave "ecology"? 
Alternatively, we could update all the "department of ecology" tags.

**Review by hollandjg — COMMENTED** [review]
I'm not sure about the department. Otherwise, looks good!

File: schemas/collaborations.yml (hunk 2 above)
Comment (RobertGemmaJr) [reply schemas/collaborations.yml:50]: Hmm... definitely looks like there isn't just an "ecology" department. I'll ask Beth and see if she has any insight!

**Comment by tdivoll** [comment]
It probably should be EEOB. Ecology, Evolution, Organisms, and their
Biology are all closely tied but there are so many specialities and
subfields within!

**Comment by RobertGemmaJr** [comment]
> It probably should be EEOB

@tdivoll I did confirm with her the department name is "Ecology, Evolution, and Organismal Biology." So the short name should be `eeob`? 

The other collaborations using the "ecology" department are `acadia` and `riddc`. Should I update those as well?

**Comment by hollandjg** [comment]
> So the short name should be `eeob`?

I think if we write "ecology-evolution-and-organismal-biology" everywhere (as you have for brainerd) and we use that everywhere, that's the clearest. (I much prefer the long name than the acronym `eeob`.)

**Comment by hollandjg** [comment]
I've made #199 to update the other cases as a PR to this one – @RobertGemmaJr feel free to merge it into here if that works for you!

**Comment by RobertGemmaJr** [comment]
> I've made #199 to update the other cases as a PR to this one 

Thank you for that! I just merged it in so this PR should be all set?

**Review by hollandjg — APPROVED** [review]
LGTM! 🚀

**Pull request merged**